x-pack/test/ingest_manager_api_integration/apis/epm/install_prerelease.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../../api_integration/ftr_provider_context';
+import { warnAndSkipTest } from '../../helpers';
+import { deletePackage } from './install_overrides';
+
+export default function ({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const dockerServers = getService('dockerServers');
+  const log = getService('log');
+
+  const pkgkey = 'prerelease-0.1.0-dev.0+abc';
+  const server = dockerServers.get('registry');
+
+  describe('installs package that has a prerelease version', async () => {
+    after(async () => {
+      if (server.enabled) {
+        // remove the package just in case it being installed will affect other tests
+        await deletePackage(supertest, pkgkey);
```

Comment:
I'd prefer to call the delete API here rather than coupling the tests to one another, but it's not a blocker

Reply:
Ah ok, I'll switch it back 👍